data/analytics/events.yml

```diff
@@ -673,7 +673,43 @@
           value: Get emails from GOV.UK
         - name: type
           value: email subscription
-
+- name: filter
+  description: Interactions that can be conducted after filtering the content using CSS instead of HTTP or API requests.
+  events:
+  - name: filter search
+    implemented: true
+    priority: low
+    description: When a user searches for a link on a page, and the search box is one that filters all the results using CSS instead of HTTP or API requests.
+    tracker: focus_loss_tracker
+    data:
+      - name: event
+        value: event_data
+      - name: event_data
+        value:
+        - name: event_name
+          value: filter
+        - name: type
+          value: filter
+        - name: action
+          value: filter
+        - name: section
+  - name: filter link clicks
```

Comment:
Does the filter link click not also include all the regular link data, like href/url, text, domain, method etc.?

Reply:
Good point, thanks - will update